editor/src/components/canvas/controls/margin-controls.tsx

```diff
@@ -21,9 +21,11 @@ export const MarginControls = betterReactMemo('MarginControls', (props: MarginCo
       props.margin.left != null && props.margin.left !== 0 ? (
         <div
           key='marginLeft'
-          className='f10 fw5 tc overflow-visible'
           style={{
             background: colorTheme.canvasLayoutFillTranslucent.value,
+            fontWeight: 500,
```

Comment:
I believe this is changing the font size since `f10` was setting it to `10px`? Was that change deliberate?

Reply:
No, will add it back in. That said, I don't even know if we still _render_ these margin controls...